conf/main/mindmaps-engine.properties

```diff
@@ -48,4 +48,4 @@ backgroundTasks.time-lapse=60000
 loader.threads=0
 
 #MindmapsDB version
-project.version=0.1.0
+project.version=0.2.0-SNAPSHOT
```

Comment:
If it needs to be commonly accessed by various components then yes I think it should go in base and base kind of does represent our main version anyway.